save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/MyProjectsRating.kt

```diff
@@ -0,0 +1,80 @@
+/**
+ * Rating of projects linked to current user (where this user is added)
+ */
+
+package com.saveourtool.save.frontend.components.views.contests
+
+import com.saveourtool.save.entities.Project
+import com.saveourtool.save.frontend.externals.fontawesome.faUser
+import com.saveourtool.save.frontend.utils.*
+
+import csstype.*
+import react.*
+import react.dom.html.ReactHTML
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.h4
+import react.dom.html.ReactHTML.p
+
+import kotlinx.js.jso
+
+val myProjectsRating = myProjectsRatings()
+
+/**
+ * @return functional component
+ */
+fun myProjectsRatings() = FC<ContestListViewProps> { props ->
+    val (myProjects, setMyProjects) = useState<Set<Project>>(emptySet())
+    val getMyProjects = useDeferredRequest {
+        setMyProjects(
+            get(
+                url = "$apiUrl/projects/current-user?userId=${props.currentUserInfo?.id}",
```

Comment:
```suggestion
                url = "$apiUrl/projects/current-user${props.currentUserInfo?.id?.let { "?userId=$it" } ?: ""}",
```

Reply:
made it a little bit differently - I don't want to have a request with empty string to the backend...